### ISSUE TITLE
make my own data

### ISSUE THREAD

**Issue by kspook** [Sierkinhane/Crnn_Ctpn_Unified#2]
I tried to train with my data. but I couldn't executed.
please help me to solve below problem.

kspook@lnxsmallv4-07101538:/data/home/kspook/Crnn_Ctpn_Unified/end2endDec_Rec/train_crnn$ python crnn_main.py --trainroot ../../dataset_preprocess/ --valroot ../../dataset_preprocess/
Namespace(cuda=False, trainroot='../../dataset_preprocess/', valroot='../../dataset_preprocess/')
read train set
<torch.utils.data.dataloader.DataLoader object at 0x7f0653049ac8>
read test set
test data set
cnn and rnn
loss averager
1
length train loader
Corrupted image for 2
Corrupted image for 4

******** my git as follows ***
https://github.com/kspook/Crnn_Ctpn_Unified.git

**Comment by Sierkinhane** [comment]
This repository I won't update. You can refer to the other repository

**Comment by Sierkinhane** [comment]
https://github.com/Sierkinhane/crnn_chinese_characters_rec
